server/app/lib/fusor/validators/deployment_validator.rb

```diff
@@ -138,6 +138,29 @@ def validate_nfs_mount(deployment, address, path)
           return
         end
 
+        # Check ownership of NFS share. /tmp/fusor-test-mount-<deployment id>
+        # is defined in safe-mount.sh as the temporary mount point
+        cmd = "stat -c '%u' /tmp/fusor-test-mount-'#{deployment.id}'"
+        status, output = Utils::Fusor::CommandUtils.run_command(cmd)
```

Comment:
I think rubocop might complain about this. If it does, just change this to one of these:

```
_, output = Utils::Fusor::CommandUtils.run_command(cmd)
output = Utils::Fusor::CommandUtils.run_command(cmd).last
```